app/concepts/user/operation/list_remove_item.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module User::Operation
+  class ListRemoveItem < Trailblazer::Operation
+    step :remove_movie_from_list
```

Comment:
not related to graphql:
it's better to assign model in a separate step: `step :model, fail_fast: true`
if it fails, it will be handled in `OperationMatcher #not_found` -> `MatchOperationResult #not_found` and the FE will get a standard not_found message